db/schema.rb

```diff
@@ -60,7 +60,7 @@
     t.integer "team_min"
     t.integer "team_max"
     t.integer "total_late_days"
-    t.integer "lateness_config_id", default: 0
+    t.integer "lateness_config_id", default: 0, null: false
```

Comment:
That's fine.  I'm not quite sure how it ever got dropped: it says `null: false` in the initial migration, and nothing else mentions it, so where could it have gone?  It got lost in https://github.com/CodeGrade/bottlenose/commit/04f258f1af46587d90d4ecf708a29267b05531c3, for no apparent reason.